catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -47,16 +47,21 @@ const SearchForm = ({ ariaDescribedBy, compact }: Props) => {
     workType,
     _queryType,
     setQueryType,
-    _dateFrom,
+    _isFilteringBySubcategory,
     _dateTo,
+    _dateFrom,
   } = useContext(CatalogueSearchContext);
 
   // This is the query used by the input, that is then eventually passed to the
   // Router
   const [inputQuery, setInputQuery] = useState(query);
+  const [shouldShowFilters, setShouldShowFilters] = useState(false);
   const searchInput = useRef(null);
-  const [inputDateFrom, setInputDateFrom] = useState(_dateFrom);
-  const [inputDateTo, setInputDateTo] = useState(_dateTo);
+
+  useEffect(() => {
+    // FIXME: not this
+    setShouldShowFilters(Boolean(Router.pathname === '/works' && query));
```

Comment:
Is this so it doesn't show on the work page?

Reply:
Yes – and not on the works landing page.